webpack.config.js

```diff
@@ -226,13 +226,14 @@ module.exports = async (env, argv) => {
 
   const host = process.env.HOST_IP || env.localDev || env.remoteDev ? "hubs.local" : "localhost";
 
-  // Remove comments from .babelrc
-  const babelConfig = JSON.parse(
-    fs
-      .readFileSync(path.resolve(__dirname, ".babelrc"))
-      .toString()
-      .replace(/\/\/.+/g, "")
-  );
+  const legacyBabelConfig = {
```

Comment:
Do you want to delete `.babelrc` and are there some settings missing? (e.g. the `"react-intl"` plugin and some of the properties of `babel/env`)

Reply:
`.babelrc` is still used for all other source files. The env options here are the crucial difference. Specifying a target of IE 11 gives us the lowest possible baseline for the browsers we need to support and we only want to apply that to support.js